doc/jsk_pcl_ros/nodes/extract_indices.rst

```diff
@@ -0,0 +1,56 @@
+ExtractIndices
+==============
+
+What Is This
+------------
+
+.. figure:: images/extract_indices.png
+   :alt: 
+
+ExtractIndices extracts point cloud with input point indices.
```

Comment:
Write about differencet between `jsk_pcl/ExtractIndices` and `pcl/ExtractIndices`.